src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,253 @@
+"""Use dnstwist to fuzz domain names and cross check with a blocklist."""
+# Standard Python Libraries
+import datetime
+import json
+import pathlib
+import traceback
+
+# Third-Party Libraries
+import dnstwist
+import dshield
+import psycopg2.extras as extras
+import requests
+
+# cisagov Libraries
+from pe_reports import app
+
+from .data.pe_db.db_query_source import (
+    addSubdomain,
+    connect,
+    getDataSource,
+    getSubdomain,
+    org_root_domains,
+    query_orgs_rev,
+)
+
+date = datetime.datetime.now().strftime("%Y-%m-%d")
+LOGGER = app.config["LOGGER"]
+
+
+def checkBlocklist(dom, sub_domain_uid, source_uid, pe_org_uid, perm_list):
+    """Cross reference the dnstwist results with DShield blocklist."""
+    malicious = False
+    attacks = 0
+    reports = 0
+    if "original" in dom["fuzzer"] or "dns_a" not in dom:
+        return None, perm_list
+    else:
+        if str(dom["dns_a"][0]) == "!ServFail":
+            return None, perm_list
+        LOGGER.info(str(dom["dns_a"][0]))
+
+        # Check IP in blocklist.de API
+        response = requests.get(
+            "http://api.blocklist.de/api.php?ip=" + str(dom["dns_a"][0])
+        ).content
+
+        if str(response) != "b'attacks: 0<br />reports: 0<br />'":
+            malicious = True
+            attacks = int(str(response).split("attacks: ")[1].split("<")[0])
+            reports = int(str(response).split("reports: ")[1].split("<")[0])
+
+        # Check IP in DShield API
+        try:
+            results = json.loads(dshield.ip(domain_a, return_format=dshield.JSON))
```

Comment:
This line won't work as is unless you accept my [earlier suggestion](https://github.com/cisagov/pe-reports/pull/259/files#r1021779698) to create the `domain_a` variable.

Reply:
Created the domain_a variable

Reply:
Latest Commit Solves Issue